### PR TITLE
Fixed questions page

### PR DESCRIPTION
**Pull request by dpionk**
checkboxes do not stay checked, this will be fixed when test is completedand we will have some answers

### CODE REVIEW

File: skin-resque-frontend/src/pages/SkintypeTest/Questions/Questions.js [hunk 1]
```diff
@@ -10,28 +10,86 @@ function Questions({ currentQuestion, numberOfQuestions }) {
...
-								<div className='answer'>
-									<Checkbox checked={false}>Ut enim ad minim veniam, quis nostrud exercitation ullamco</Checkbox>
-								</div>
-								<div className='answer'>
-									<Checkbox checked={false}>Ut enim ad minim veniam, quis nostrud exercitation ullamco</Checkbox>
-								</div>
-								<div className='answer'>
-									<Checkbox checked={false}>Ut enim ad minim veniam, quis nostrud exercitation ullamco</Checkbox>
-								</div>
+							{answersToDisplay && answersToDisplay}
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/pages/SkintypeTest/Questions/Questions.js:92]: Why the double the `answersToDisplay`?

**Review by kbojarowska — CHANGES_REQUESTED** [review]
image
Everything is too big, not using svg background

**Comment by dpionk** [comment]
@kbojarowska  because the fix was not including using svg background. It was about question number and checkboxes.

**Comment by kbojarowska** [comment]
> @kbojarowska because the fix was not including using svg background. It was about question number and checkboxes.

we were talking about changing all of the skintype test backgrounds to svg

**Comment by dpionk** [comment]
Then I would include this change with different issue

**Comment by dpionk** [comment]
This was corresponding with issue where there were two things to fix and they were not about svg background

**Comment by dpionk** [comment]
I deleted pagination as it was using links which are not compatible. I will be working on skin type test right now.

**Review by MrSpaceMan1 — APPROVED** [review]

**Review by kbojarowska — APPROVED** [review]

**Pull request merged**